### PR TITLE
Add info about order of touches in the array

### PR DESCRIPTION
**Pull request by advitiya7**
### Description
added clarity about the the `touchEvent` command and its working when array is used.

### Motivation

i really wanted to contribute to open source as it can help me in improving my problem solving things and i get to know how the things in the industry works.

### Related issues and pull requests
fixed issue #20811

### CODE REVIEW

File: files/en-us/web/api/touchevent/touches/index.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ target element was at {{domxref("Element/touchstart_event", "touchstart")}} time
 You can think of it as how many separate fingers are able to be identified as touching
 the screen.
 
+One thing to note that the touches in `TouchEvent.touches[i]` doesn't neccessarily gives you i-th index of the array.Merely,the array is not ordered predictably and it depends upon implementation. A      developer cannot just **assume** a specific order,like the
+i-th element in the array being the i-th touch that happened,and the order in which touches
+happen is independent of the order of the list.To determine the latter,developers should rely
+on the the `touch` object IDs.
+
+
+
```
Comment (teoli2003) [inline files/en-us/web/api/touchevent/touches/index.md:24]:
```suggestion
> **Note:** Touches inside the array are not necessarily ordered by order of occurrences (the
i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.
```

File: files/en-us/web/api/touchevent/touches/index.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ target element was at {{domxref("Element/touchstart_event", "touchstart")}} time
 You can think of it as how many separate fingers are able to be identified as touching
 the screen.
 
+> **Note:** Touches inside the array are not necessarily ordered by order of occurrences (the
+i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.
```
Comment (teoli2003) [inline files/en-us/web/api/touchevent/touches/index.md:19]:
```suggestion
i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.

```

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by schalkneethling** [comment]
Welcome to the community, @advitiya7  🎉 Thank you for your contribution. If you are interested in more ways to contribute to MDN Web Docs, please have a look at our [contributor task board](https://github.com/orgs/mdn/projects/25). If you have any questions or need help finding something to work on, please ask for help in our [community discussions](https://github.com/orgs/mdn/discussions). We look forward to more of your contributions. 🙌

**Comment by advitiya7** [comment]
@schalkneethling thanks a lot!! I  am looking to contribute more to mdn and learn from the community.

**Comment by hamishwillee** [comment]
Thanks @advitiya7 
I can see you have asked about a bunch of issues that you might help with. Perhaps limiting yourself to just one or two in the first case might be a good idea, so we can mentor you through our ways-of-working. I can probably help you with https://github.com/mdn/content/issues/25342 if that one is of interest.

**Comment by advitiya7** [comment]
@hamishwillee i am sorry and i will keep that in check in future.

**Comment by hamishwillee** [comment]
There's no need to be sorry @advitiya7 - your contribution is welcome. I'm just suggesting we start a little slower.

**Comment by advitiya7** [comment]
@hamishwillee definitely i will keep that in my mind thanks again for the guidance

**Pull request merged**